token/token.go

```diff
@@ -25,29 +25,32 @@ const (
 
 	Ident
 
+	LineTerminator
```

Comment:
I know 'line terminator' is how es5 defines it, but I think this name does not follows the naming convention of tokens used until now. What do you think of `Newline`, `LF` (line terminator includes line feed, but I dont think we need to be too generic), `Linesep`, `LineSep`, `CRLF` (carriege return and line feed, but ignoring others in the name).

I feel like the parser depends a lot on this token, typing 'LineTerminator' every time could be tedious for a very simple and intuitive token. I vote for LF or Linesep.

Reply:
I think `Linesep` is more explicit than `LF`.